vchernyshoff/homework_5/main.py

```diff
@@ -0,0 +1,168 @@
+def double_input(input_data):
+    """Doubles input"""
+    return input_data * 2
+
+
+def strip_comments(input_string, markers):
+    """
+    Complete the solution so that it strips all text that follows any of a set of comment markers passed in.
+    Any whitespace at the end of the line should also be stripped out.
+
+    Example:
+
+    Input string:
+
+    "
+    apples, pears # and bananas
+    grapes
+    bananas !apples
+    "
+
+    Markers: ["#", "!"]
+
+    The output expected would be:
+
+    "
+    apples, pears
+    grapes
+    bananas
+    "
+
+    result = strip_comments("apples, pears # and bananas\ngrapes\nbananas !apples", ["#", "!"])
+    # "apples, pears\ngrapes\nbananas"
+    """
+    new_strings = ''
+    strings = input_string.split('\n')
+    for string in strings:
+        new_string = string
+        for next_symbol in string:
+            if next_symbol in markers:
+                new_string = string.split(next_symbol)[0]
+        new_strings += new_string.strip() + '\n'
+    new_strings = new_strings[:-1]  # cut last \n
```

Comment:
could you remind me to discuss that part on our next call, please

Reply:
Sure